lib/spack/docs/developer_guide.rst

```diff
@@ -299,6 +315,198 @@ Conceptually, packages are overloaded.  They contain:
 Stage objects
 -------------
 
+
+.. _writing-analyzers:
+
+-----------------
+Writing analyzers
+-----------------
+
+To write an analyzer, you can either include it in one of the files in the
+analyzers module folder (e.g., if it's associated with an application
```

Comment:
where's the analyzers module folder (also should it be "directory"?)

Reply:
It's at `lib/spack/spack/analyzers`, I'll add that here and change folder to directory.